src/bootstrap.coffee

```diff
@@ -11,9 +10,6 @@ _ = require 'lodash'
 deviceConfig = require './device-config'
 userConfig = {}
 
-DuplicateUuidError = (err) ->
```

Comment:
Looks like we won't be catching this anymore... do we want to handle this case in the API? (make the register endpoint idempotent if the uuid exists and it's a valid provisioning key?)